frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -33,7 +33,7 @@ export const perspective = {
   switchTo: (perspectiveName: switchPerspective) => {
     nav.sidenav.switcher.changePerspectiveTo(perspectiveName);
     app.waitForLoad();
-    if (switchPerspective.Developer) {
+    if (perspectiveName === switchPerspective.Developer) {
```

Comment:
Any way we are passing the enum inside the if condition, do we really need === operator

Reply:
@makambalaji I don't understand what you mean by this.
Before my change, the condition looked like:
```
if (switchPerspective.Developer) {
```
which is IMHO allways true. This seems like a bug to me. It looks like the intended behavior was to go inside this `if` only when the `perspectiveName` is `Developer`, which my change should ensure.